ssoadmin/client.go

```diff
@@ -510,3 +510,26 @@ func (c *Client) IdentitySources(ctx context.Context) (*types.IdentitySources, e
 
 	return &res.Returnval, nil
 }
+
+func (c *Client) GetDefaultDomains(ctx context.Context) (*[]string, error) {
```

Comment:
minor, but looks like this should return `[]string` rather than `*[]string`. That's what we do elsewhere in govmomi for list types, and Go APIs in general do the same for list types.

Reply:
Done